tmt/cli.py

```diff
@@ -320,6 +320,16 @@ def main(
 @environment_options
 @verbosity_options
 @force_dry_options
+@option(
+    '--feeling-safe',
+    is_flag=True,
+    help="""
+         WARNING: with this option, tmt would be allowed to make
+         potentially dangerous actions. For example, some metadata
+         keys may cause scripts being executed on the runner.
+         Do not use this option unless you trust metadata consumed
+         by tmt, or unless you know what you are doing.
+         """)
 def run(context: Context, id_: Optional[str], **kwargs: Any) -> None:
```

Comment:
Shouldn't this be a main `tmt` command option? Similar as the `--context` this seems to be a generic choice. And I guess there will be use cases for other subcommands as well, for example `tmt test import` or `tmt plan show` for remote plans.